geode-core/src/main/java/org/apache/geode/cache/EvictionAttributes.java

```diff
@@ -541,7 +544,7 @@ public static EvictionAttributes createLIFOMemoryAttributes(int maximumMegabytes
   }
 
   public boolean isEmpty() {
-    return getAction() == EvictionAction.NONE && getAlgorithm() == EvictionAlgorithm.NONE;
+    return getAction() == EvictionAction.NONE;
```

Comment:
why we don't care about algorithm anymore here?

Reply:
We had a test case where the algorithm was provided but the intended behavior was no eviction (since action was none).